### PR TITLE
Nerfs Warding bonus armor and Frenzy speed bonus

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
People were complaining about warding having dumb scaling, so I made it max 5 than 25
I later nerfed Frenzy because it made benos too fast, and pheromones being so performance-enhancing is poopoo. don't merge/tm without <https://github.com/tgstation/TerraGov-Marine-Corps/pull/3774>

## Why It's Good For The Game
warding/frenzy being so good kind of breaks the game

## Changelog
:cl:

balance: warding nerfed
/:cl:

### CODE REVIEW

**Comment by psykzz** [comment]
Putting a hold on this Pr until we resolved the chem PR and the other massive xeno PR.

File: code/modules/mob/living/carbon/xenomorph/life.dm [hunk 1]
```diff
@@ -224,7 +224,7 @@
 	recovery_new = 0
 	armor_pheromone_bonus = 0
 	if(warding_aura > 0)
-		armor_pheromone_bonus = warding_aura * 5 //Bonus armor from pheromones, no matter what the armor was previously.
+		armor_pheromone_bonus = warding_aura * 1 //Bonus armor from pheromones, no matter what the armor was previously.
```
Comment (ma44) [inline code/modules/mob/living/carbon/xenomorph/life.dm:227]: bruh

**Comment by drawsstuff** [comment]
Christ, just remove pheromones at this point

**Comment by AdamElTablawy** [comment]
armor_pheromone_bonus = warding_aura * 1 //Bonus armor from pheromones, no matter what the armor was previously.

so it literally does nothing at its weakest level then

**Comment by terranaut1** [comment]
>warding/frenzy being so good kind of breaks the game
why?

**Comment by terranaut1** [comment]
> Christ, just remove pheromones at this point

Improve, don't remove

**Comment by Pariah919** [comment]
I'm closing this for the meantime, Once the big changes with sundering are threw I'll reopen it.

**Pull request closed without merging**